pkg/deploy/generator/scripts.go

```diff
@@ -15,3 +15,6 @@ var scriptGatewayVMSS []byte
 
 //go:embed scripts/rpVMSS.sh
 var scriptRpVMSS []byte
+
+//go:embed scripts/commonVMSS.sh
+var scriptCommonVMSS []byte
```

Comment:
`commonVMSS.sh` is a massive file - over a thousand lines!

Could we do the following:
- Refactor `commonVMSS.sh` into separate scripts
- Have a function that concats all those scripts into a byte slice called `scriptCommonVMSS`
- Then append the other scripts (ex: `rpVMSS.sh`) for deployment.

This way we can make `commonVMSS.sh` more readable.

Reply:
Appending will not work, because the functions won't be declared in `rpVMSS.sh`.
The functions would be called before they have been created.

Reply:
@niontive I've broken `commonVMSS.sh` up into 4 different `util-*.sh` files.
I'm testing it in INT now.